resources/views/participant-detail.blade.php

```diff
@@ -16,6 +16,11 @@
             <div class="col">
                 <h3>{{ $participant->scout_name }}</h3>
                 @if (isset($participant->group))<h5>{{ $participant->group }}</h5>@endif
+                @if (isset($participant->freetext))<p>{{ $participant->freetext }}</p>@endif
+                <a href="{{route('admin.participants.edit', ['course' => $course->id, 'participant' => $participant->id])}}">
```

Comment:
De Link isch aktuell weg de Margin vom `<p>` no chli wiit une, und chlebt bitz am druf folgende Text. Vilicht chönntsch de Link is `<p>` Tag ine verschiebe, und mit `class="ml-3"` eifach bitz horizontale Abstand zum Text mache?